docs/get-started/getting-started-workload-qualification.md

```diff
@@ -0,0 +1,141 @@
+If you plan to convert existing Spark workload from CPU to GPU, it is highly recommended that you do gap analysis 
+to figure out if there are any unsupported features such as functions, expressions, data types, data formats.
+After that we will know which workload is more suitable to GPU, which is called "workload qualification" here.
+
+If there are certain operators which can not run on GPU due to the current limitations, they will fallback to CPU mode.
+As a result, it may incur some performance overhead because of host memory <=> GPU memory transfer.
+However it is not critical to ensure all operations are supported by the GPU and significant performance benefits can 
+be gained before all operations are fully supported by the GPU. It all depends on how critical the portion that is 
+executing on the CPU is to the overall performance of the query.
+
+This article is to help you get familiar with the best practice and different tools we provide on how to do gap analysis 
+and workload qualification.
+
+## 1. Qualification and Profiling tool
+
+### Requirement
+
+- Access to Spark event logs from Spark 2.x or 3.x
+- Spark 3.0.1+ jars
+- `rapids-4-spark-tools` jar
+
+### How to use
+
+If you have Spark event logs from prior runs of the applications on Spark 2.x or 3.x, you can use the
+[Qualification tool](../spark-qualification-tool.md) and [Profiling tool](../spark-profiling-tool.md) to analyze them.
+The qualification tool outputs the score, rank and some of the potentially not-supported features for each Spark application.
+Its output can help you focus on the top N Spark applications which are SQL heavy applications. 
+
+The profiling tool outputs SQL plan metrics and also prints out actual query plans to provide more insights.
+The following example profiling tool output for a specific Spark application shows that it has a query with a large 
+`HashAggregate` and `SortMergeJoin`. Those are indicators for a good candidate for the RAPIDS Accelerator.
+
+```
++--------+-----+------+----------------------------------------------------+-------------+------------------------------------+-------------+----------+
+|appIndex|sqlID|nodeID|nodeName                                            |accumulatorId|name                                |max_value    |metricType|
++--------+-----+------+----------------------------------------------------+-------------+------------------------------------+-------------+----------+
+|1       |88   |8     |SortMergeJoin                                       |11111        |number of output rows               |500000000    |sum       |
+|1       |88   |9     |HashAggregate                                       |22222        |number of output rows               |600000000    |sum       |
+```
+
+Since the two tools are only analyzing Spark event logs, it can not provide very accurate gap analysis information.
+However it is very convenient because you do not need a CPU or GPU Spark cluster to run them.
+
```

Comment:
so the other thing that I think is missing is really how the user interprets this output.  Again this kind of goes along with Chris' doc perhaps but they run this tool then what?  I'm fine with that being separate and perhaps going into each of the docs, but we may also want to say something like,... run qualification tool get score, if things look ok, then run profiling tool to get query plans, or skip to explainPotentialGPUPlan if possible, etc..  If we aren't doing that here can you please file a separate issue to follow up once we have discussed with Chris and team.

Reply:
For profiling tool, i gave one example on how to interprets the output such as looking at SQL plan metrics.
For qualification tool, how about we just mention the `unsupported data types`, `udf` etc which could be related to gap analysis?

Reply:
yeah that's fine for now.

Reply:
I just added:
```
The qualification tool outputs the score, rank and some of the potentially not-supported features for each Spark application.
For example, the CSV output can output `Potential Problems`, `Unsupported Read File Formats and Types`, and
`Unsupported Write Data Format` which are the indication of some not-supported features.
Its output can help you focus on the top N Spark applications which are SQL heavy applications.
```